modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts

```diff
@@ -186,6 +186,16 @@ export class DocumentLibraryPage {
 		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
 	}
 
+	async goToPermission(entryTitle: string) {
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {name: 'Permissions'}),
+			trigger: this.page
+				.locator(`.card-body:has-text('${entryTitle}')`)
+				.getByLabel('More actions'),
+		});
+	}
+
```

Comment:
Can we refactor this to to the same approach that we implemented in journal?

We can implement `goToActions` instead of hardcoded the `{name: 'Permissions'}`